### PR TITLE
Removed "Load from .env" section

### PR DESCRIPTION
**Pull request by Zeesky-code**
*  What - Solving PR [Docs: remove the "Load from .env" section from the "Configure your Server" doc page ](https://github.com/medusajs/medusa/issues/2286)

* Why - The "Load from .env" section in the [Configure your Server documentation](https://docs.medusajs.com/usage/configurations) is no longer relevant now that it is included by default in the starter.

*  How - Following the guideline in the issue made the changes to `medusa/docs/content/usage/configurations.md`

* Testing - No test written because it's a markup file.

### CODE REVIEW

File: docs/content/usage/configurations.md [hunk 1]
```diff
@@ -24,50 +24,7 @@ This change in how environment variables are loaded was introduced in version 1.
...
-
-:::note
-
-`dotenv` should be available to use in your Medusa server project without the need to install it. However, if it’s not available you can install it with the following command:
-
-```npm2yarn
-npm install dotenv --save
-```
-
-:::
```
Comment (shahednasser) [inline docs/content/usage/configurations.md:28]: Can you remove this empty line? There are now two empty lines above "Database Configuration"

Comment (Zeesky-code) [reply docs/content/usage/configurations.md:28]: I have removed it.

Comment (shahednasser) [reply docs/content/usage/configurations.md:28]: Did you commit the change? I can't see it

Comment (Zeesky-code) [reply docs/content/usage/configurations.md]: Sorry, I've done that now :)

File: docs/content/usage/configurations.md [hunk 2]
```diff
@@ -25,7 +25,6 @@ This change in how environment variables are loaded was introduced in version 1.
 :::
```
Comment (shahednasser) [inline docs/content/usage/configurations.md:27]: Sorry there's one more line 😅 Can you remove it as well?

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks! @Zeesky-code 🚀

**Pull request merged**